Include/pyport.h

```diff
@@ -505,13 +505,16 @@ extern "C" {
 /* Py_DEPRECATED(version)
  * Declare a variable, type, or function deprecated.
  * Usage:
- *    extern int old_var Py_DEPRECATED(2.3);
- *    typedef int T1 Py_DEPRECATED(2.4);
- *    extern int x() Py_DEPRECATED(2.5);
+ *    Py_DEPRECATED(2.3) extern int old_var;
```

Comment:
Did you test this?

Reply:
Yes.

Reply:
Sorry, by "yes" I meant that this example compiles, but it is actually a useless example and should be removed.